roles/openshift_web_console/defaults/main.yml

```diff
@@ -8,11 +8,11 @@ __console_config_file: "console-config.yaml"
 openshift_web_console_image_dict:
   origin:
     prefix: "docker.io/openshift/origin-"
-    version: "{{ openshift_image_tag }}"
+    version: "{{ openshift_web_console_version | default(openshift_image_tag) }}"
```

Comment:
This would be a circular reference.

Users can already set openshift_web_console_version; it defaults to openshift_image_tag.  See line 19 below.